### ISSUE TITLE
discourse update script broken

### ISSUE THREAD

**Issue by yayayayaka** [NixOS/nixpkgs#321499]
### Describe the bug
The discourse [`update.py`](https://github.com/NixOS/nixpkgs/blob/c380d4f0612f594588ed9f73606063e24ed06d9e/pkgs/servers/web-apps/discourse/update.py#L2) script depends on `nix-universal-prefetch`, which was removed by 578d7e717848de3259baa52262a9267a9cdf6277 in #321436.

### Steps To Reproduce
1. `pkgs/servers/web-apps/discourse/update.py update`

### Expected behavior
The update scripts succeeds.

### Notify maintainers
@talyz 

### Screenshots
```
error:
       … while calling the 'derivationStrict' builtin
         at /builtin/derivation.nix:1:208:
       … while evaluating derivation 'shell'
         whose name attribute is located at /nix/store/r88r99f9j21ixqjxxw60wq3y4i258cqz-source/pkgs/stdenv/generic/make-derivation.nix:331:7

       … while evaluating attribute 'buildInputs' of derivation 'shell'
         at /nix/store/r88r99f9j21ixqjxxw60wq3y4i258cqz-source/pkgs/stdenv/generic/make-derivation.nix:378:7:
          377|       depsHostHost                = elemAt (elemAt dependencies 1) 0;
          378|       buildInputs                 = elemAt (elemAt dependencies 1) 1;
             |       ^
          379|       depsTargetTarget            = elemAt (elemAt dependencies 2) 0;

       (stack trace truncated; use '--show-trace' to show the full trace)

       error: The nix-universal-prefetch package was dropped since it was unmaintained.
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by Mic92** [comment]
Could `nix-update` replace `nix-universal-prefetch` in these instances?

**Comment by FliegendeWurst** [comment]
Fixed in #324349
